core/src/main/java/org/apache/calcite/sql/SqlUtil.java

```diff
@@ -1238,6 +1238,17 @@ public static boolean containsIn(SqlNode node) {
     return containsCall(node, callPredicate);
   }
 
+  /**
+   * Determine presence at least one node with {@link SqlKind#DEFAULT} type.
+   *
+   * @param node AST tree
+   */
+  public static boolean containsDefault(SqlNode node) {
+    final Predicate<SqlCall> callPredicate = call ->
```

Comment:
I saw other containsXXX method used `call.getOperator() instanceof XXXOperator`.

Could we use `call -> call.getOperator() instanceof SqlDefaultOperator` here to keep the same behavior with others?

Reply:
"eq" comparator is faster than instanceof, so i prefer this form

Reply:
Get your point.

You think in terms of performance, I think in terms of code style,I don't know the performance difference between them,but it is ok for me.